### ISSUE TITLE
Interactive control are not keyboard accessible.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5738]
### What is the PWD impact?

When interactive controls are not keyboard accessible, users who rely on keyboard navigation—including those with motor impairments, screen reader users, or individuals who cannot use a mouse—are unable to operate the controls.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Audio'.
3. Navigate to bot response.
4. Observe the issue that Interactive audio control are not keyboard accessible. 

### What do you expect?

Keyboard focus should move to the audio control and should be actionable with enter key.

### What actually happened?

Interactive audio control is not keyboard accessible. 

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/e7ca36da-411e-4494-b56f-6d360ca317f0

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/keyboard

### Adaptive Card JSON

```json

```
